tests/manager/core/classifiers/test_classifier.py

```diff
@@ -429,7 +429,7 @@ def test_no_children_or_ya_signal_from_distributor_implies_book_is_for_adults(
         # YA. classifier.audience_weights[AUDIENCE_ADULT] will be set
         # to 500.
         i = data.identifier
-        source = DataSource.lookup(session, DataSource.OVERDRIVE)
+        source = DataSource.lookup(session, DataSource.OVERDRIVE, autocreate=True)
```

Comment:
This is just done for typing benefit, since setting autocreate make sure we always get a `Datasource` back rather then `Datasource | None`